packages/web-components/README.md

```diff
@@ -45,7 +45,7 @@ environment that `agoric-wallet-connection` uses:
 
 ```js
 // Ensure this is imported before anything else in your project.
-import './install-ses-lockdown.js';
+import './install-demo-ses-lockdown.js';
```

Comment:
We probably just want to remove this whole section, it's not a great example. Perhaps we should just link to https://github.com/Agoric/dapp-psm/blob/main/src/installSesLockdown.ts as an example (oh, looks like we need to update the `errorTaming` there too).